### ISSUE TITLE
Pods-resources.sh badly generated during pod install

### ISSUE THREAD

**Issue by lauracpierre** [CocoaPods/CocoaPods#3739]
Hi everyone, 

Just tried to install HockeyApp in my app and overall it worked just fine. That being said I encountered one problem while building. 
My project name is 'ios' (don't ask...) so the generate shell script in which there is an issue is: 
Pods-ios-resource.sh

When trying to rync the bundle, it failed saying the bundle couldn't be found. I tracked it down to here:

**Wrong (but generated for every pod install):**
```sh
if [[ "$CONFIGURATION" == "Release" ]]; then
  install_resource "${BUILT_PRODUCTS_DIR}/HockeySDKResources.bundle"
fi
```

**Correct: (changed manually):**
```sh
if [[ "$CONFIGURATION" == "Release" ]]; then
  install_resource "${BUILT_PRODUCTS_DIR}/Pods-ios/HockeySDKResources.bundle"
fi
```

Someone the script is generated with the wrong path. 
Here are more details that might help: 
- ios project was created before the first pod install
- ios project and Pods project are in the same workspace
- Cocoapod version used is 0.37.2
- We are using swift and many other frameworks

Thanks a lot!

**Comment by neonichu** [comment]
Could you please retry this with the 0.38 beta?

**Comment by lauracpierre** [comment]
Just tried with 0.38.0.beta.1 but still seeing the same issue

**Comment by segiddins** [comment]
@lauracpierre could you share an example project we can use to investigate the issue?

**Comment by lauracpierre** [comment]
Sure thing

I originally created the project without Cocoapod, and then added the podfile and ran pod install which created the workspace and Pod project.
Here is our current pod file


platform :ios, '8.0'

# ignore all warnings from all pods
inhibit_all_warnings!

use_frameworks!

target 'ios' do
    pod 'libPusher', '~> 1.6'
    pod 'Reachability', :git => 'https://github.com/lauracpierre/Reachability.git'
    pod 'Alamofire'
    pod 'SwiftyJSON', :git => 'https://github.com/SwiftyJSON/SwiftyJSON.git'
    pod 'Dollar'
    pod 'Cent'
    pod 'SlackTextViewController'
    pod 'HanekeSwift'
    pod 'HockeySDK', :subspecs => ['CrashOnlyLib']
end

We have extra schemes but the problem occur on all of them.

Anything else you need to know?

**Comment by lauracpierre** [comment]
Hi Guys, 
do you need more details from me?

Thanks !

**Comment by amorde** [comment]
I tried to reproduce this by creating a new project named "ios" with your Podfile, running `pod install` and then running the project. The project runs successfully with no errors, so I'm not sure what exactly the issue is.

I inserted a call to `ls` in the resource script like so:
```shell
if [[ "$CONFIGURATION" == "Release" ]]; then
  ls "${BUILT_PRODUCTS_DIR}/"
  install_resource "${BUILT_PRODUCTS_DIR}/HockeySDKResources.bundle"
fi
```
This outputs:
```
~/Developer/Xcode/DerivedData/ios-aiakzchngrqoapguqxummmhxihml/Build/Products/Debug-iphonesimulator/HockeySDKResources.bundle
```

There is no `Pods-ios` directory as you have in your script - the Hockey bundle is the only thing in that directory. 

Tested on `0.38.0.beta.1`
